### PR TITLE
docs(repo): add docs for expected structure

### LINKED ISSUE #8: document input file structure
Document the expected structure for the input file.

### PR DESCRIPTION
**Pull request by blake-newman**
add documenation around the expected structure for the file input
structure with link to example

closes GH-8

### CODE REVIEW

**Review by csi-lk — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -40,3 +40,26 @@ jobs:
...
+  column?: AnnotationSourcePositionReference
+  level: AnnotationLevel
+}
+
+interface AnnotationSourcePositionReference {
+  start?: number
+  end?: number
+}
+
+type Annotations = Annotation[]
```
Comment (ycmjason) [inline README.md]: don't like this.

Comment (ycmjason) [reply README.md]: think `Annotation[]` is clear enough. `Annotations` just adds more cognitive overhead to readers.

Comment (nowy) [reply README.md]: same here! I would assume it was an object

Comment (blake-newman) [reply README.md]: In fact i can just reference the source; less overhead for maintenance

**Review by ycmjason — APPROVED** [review]

**Pull request merged**